src/porepy/models/constitutive_laws.py

```diff
@@ -3428,6 +3474,30 @@ def mechanical_stress(self, domains: pp.SubdomainsOrBoundaries) -> pp.ad.Operato
         stress.set_name("mechanical_stress")
         return stress
 
+    def combine_boundary_operators_mechanical_stress(
+        self, subdomains: list[pp.Grid]
+    ) -> pp.ad.Operator:
+        """Combine mechanical stress boundary operators.
```

Comment:
Should we explain in the documentation of these methods that they can be used to define other Robin operators? I.e., not use `mechanical_stress` for both neu and rob.